packages/preview-middleware-client/src/adp/ui/FileExistsDialog.fragment.xml

```diff
@@ -0,0 +1,33 @@
+<Dialog
+    id="fileExistsDialog"
+    xmlns:mvc="sap.ui.core.mvc"
+    xmlns="sap.m"
+    xmlns:core="sap.ui.core"
+    xmlns:f="sap.ui.layout.form"
+    title="{i18n>ANNOTATION_FILE_EXISTS}"
+    contentWidth="450px"
+    class="sapUiRTABorder"
+    type="Message"
+>
+    <content>
+        <Text text="{i18n>ANNOTATION_FILE_HAS_BEEN_FOUND}" />
+        <Text text="{/filePathFromRoot}" />
```

Comment:
This needs to be revisited in another PR. Because the file name has changed my comment is now gone. We can use a form with Label and Input that is in disabled-like state.

Reply:
sure. marking as resolved for merging